akka-cluster/src/multi-jvm/scala/akka/cluster/artery/ArterySunnyWeatherSpec.scala

```diff
@@ -1,18 +1,19 @@
 /**
  * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.cluster
+package akka.cluster.artery
```

Comment:
we probably want to keep both version as long as we support the old remoting, so a copy, or make the effort to do 2 variants of the test with 2 different configs. I think we have that in some tests already, such as those with the failure detector puppet, e.g. https://github.com/akka/akka/blob/master/akka-cluster/src/multi-jvm/scala/akka/cluster/ClientDowningNodeThatIsUpSpec.scala